src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/BookListSlice';
+
+const store = configureStore({
+  reducer: {
+    book: bookReducer,
+  },
+});
```

Comment:
- [ ] As required for the project, kindly ensure that you also import the `categoriesReducer` from `categoriesSlice` as use it as well, in the same way you used the `bookReducer` so to have 2 combined reducers. For example:
```jsx
...
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})
```